specification/iotcentral/data-plane/readme.md

```diff
@@ -10,34 +10,37 @@ To see additional help and options, run:
 ---
 
 ## Configuration
+
+### Basic Information
+
 These are the global settings for the IoT Central API.
+
 ``` yaml
 openapi-type: data-plane
-tag: package-2019-10-28-preview
+tag: package-2021-04-30-preview
```

Comment:
This indicates this is a preview with an API version of 2021-04-30-preview, but the swagger file exists in the stable folder. Is this GA or preview?